test/message/test_runner_only_tests.out

```diff
@@ -116,11 +116,73 @@ ok 11 - only = true, with subtests
   ---
   duration_ms: *
   ...
-1..11
-# tests 11
-# pass 1
+# Subtest: describe only = true, with subtests
+    # Subtest: `it` subtest 1 should run
+    ok 1 - `it` subtest 1 should run
+      ---
+      duration_ms: *
+      ...
+    # Subtest: `it` subtest 2 should run
+    ok 2 - `it` subtest 2 should run
+      ---
+      duration_ms: *
+      ...
+    1..2
+ok 12 - describe only = true, with subtests
+  ---
+  duration_ms: *
+  ...
+# Subtest: desribe only = true, with a mixture of subtests
+    # Subtest: `it` subtest 1
+    ok 1 - `it` subtest 1
+      ---
+      duration_ms: *
+      ...
+    # Subtest: `it` async subtest 1
+    ok 2 - `it` async subtest 1
+      ---
+      duration_ms: *
+      ...
+    # Subtest: `it` subtest 2 only=false
+    ok 3 - `it` subtest 2 only=false # SKIP 'only' option not set
+      ---
+      duration_ms: *
+      ...
+    # Subtest: `it` subtest 3 skip
+    ok 4 - `it` subtest 3 skip # SKIP
+      ---
+      duration_ms: *
+      ...
+    # Subtest: `it` subtest 4 todo
+    ok 5 - `it` subtest 4 todo # SKIP 'only' option not set
+      ---
+      duration_ms: *
+      ...
+    1..5
+ok 13 - desribe only = true, with a mixture of subtests
+  ---
+  duration_ms: *
+  ...
+# Subtest: subtest should run
+ok 14 - subtest should run # SKIP 'only' option not set
+  ---
+  duration_ms: *
+  ...
+# Subtest: async subtest should run
+ok 15 - async subtest should run # SKIP 'only' option not set
```

Comment:
Same thing here.

Reply:
This is the same as https://github.com/nodejs/node/pull/46604#discussion_r1105487817.

Reply:
Done via https://github.com/nodejs/node/commit/6f069ef8798cd12aafa9e5872e19d6daee4326d9.